index.yaml

```diff
@@ -113,7 +123,17 @@ entries:
     version: 3.0.0
   - apiVersion: v2
     appVersion: 1.16.0
-    created: "2022-03-03T18:42:07.563786+01:00"
+    created: "2023-11-08T12:33:12.098001Z"
+    description: A Helm chart for Kubernetes
+    digest: ddbb7da3b83053f94222d753c3a4bf4b8c639ad1c4b194880cf7ba25256a1957
+    name: hm-basic-webapp
+    type: application
+    urls:
+    - https://highmobility.github.io/helm-charts/hm-basic-webapp-2.2.0.tgz
+    version: 2.2.0
+  - apiVersion: v2
+    appVersion: 1.16.0
+    created: "2023-11-08T12:33:12.097452Z"
```

Comment:
Did it add an older version here? 🤔

Reply:
@slashmili [Fixed](https://github.com/highmobility/helm-charts/compare/0b3302d444d21ad005054ae649935aaa111f0542..da48d11625b8da36c9977f3c769b20b73097f7f0) i guess?